helpers/auth_x11.c

```diff
@@ -77,8 +81,86 @@ int prompt_timeout;
 //! Extra line spacing.
 #define LINE_SPACING 4
 
-//! Whether password display should hide the length.
-int paranoid_password;
+//! Deprecated flag for setting Whether password display should hide the length.
+int paranoid_password_flag;
```

Comment:
No longer needs to be global; please move these variables into main() now.

Reply:
Done in 45e0926.